src/components/grid-aware/StatsBlock/StatsBlock.stories.jsx

```diff
@@ -3,30 +3,35 @@ import StatsBlock from "./StatsBlock";
 
 export default {
   title: "Grid-Aware/StatsBlock",
+  subTitle: "Optional Subtitle",
```

Comment:
Nit: Could you make the "t" in "subtitle" lowercase? This would make it match everywhere else the prop name is spelled.